routes/index.js

```diff
@@ -28,13 +30,43 @@ router.get('/', async function(req, res, next) {
 
   } catch(err) {
     error = err;
+    console.warn(err);
   }
 
   res.render('index', {
     statusData: statusData,
+    statusUrl, statusUrl,
```

Comment:
statusUrl: statusUrl ??

Reply:
Could you clarify what you're asking here. Are you wondering what `statusUrl` is used for? see https://github.com/openactive/conformance-status-page/pull/10/files#diff-0fee1be8218cc43a268be5a23c313f2fR46

```html
   <p><a class="btn btn-secondary" href="{{statusUrl}}">Download as JSON</a></p> 
```

Reply:
Nope, I'm asking why the code is "statusUrl, statusUrl" and not "statusUrl: statusUrl" - comma not colon -  is this a JS thing I don't know?

Reply:
No that's a typo, thanks will fix